docs/roman/references_general/references_general.rst

```diff
@@ -76,20 +72,12 @@ documentation on each reference file.
 +--------------------------------------------------+---------------------------------------------+
 | :ref:`FLAT <flat_reffile>`                       | :ref:`flatfield <flatfield_step>`           |
 +--------------------------------------------------+---------------------------------------------+
-| :ref:`GAIN <gain_reffile>`                       | :ref:`jump_detection <jump_step>`           |
```

Comment:
Please restore this. The `GAIN` reference file is still used by the ramp step.

Reply:
This is still not correct. The table is not formatted and `GAIN` and `READNOISE` are missing from the lower tables:
https://roman-pipeline--1534.org.readthedocs.build/en/1534/roman/references_general/references_general.html#reference-file-types

Reply:
@ddavis-stsci would you address this issue?